<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commandlsit</class>
 <widget class="QWidget" name="commandlsit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>971</width>
     <height>551</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>770</width>
     <height>490</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1151</width>
     <height>561</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
